dns/scutil/config.go

```diff
@@ -81,6 +82,8 @@ func readMacOSDNS(ctx context.Context, getSCUtilDNS scutilExecutor) (Config, err
 			}
 		case strings.Contains(key, "options"):
 			currentResolver.MulticastDNS = strings.Contains(value, "mdns")
+		case strings.Contains(key, "port"):
+			currentResolver.Port = value
```

Comment:
Let's store the port as an `int`, a bit like `InterfaceIndex` and `Order`.

Can you add an int parser here? Looks like the same logic from `order` could be reused here.

Reply:
I have updated my PR to use an int instead. if you have an elegant way to set the default port 53, please let me know.